### PR TITLE
Lockdown installation metadata for installation

### PR DESCRIPTION
**Pull request by HLeithner**
Remove the following xml files from translation:

- version
- creationDate
- author
- copyright
- license

### CODE REVIEW

**Comment by ot2sen** [comment]
The <author> field should be valid and probably not locked. Copyright and Author right is different. The author field acknowledges the work provided by the translation teams.

**Comment by zero-24** [comment]
> The 'author' field should be valid and probably not locked. Copyright and Author right is different. The author field acknowledges the work provided by the translation teams.

We (German) have been told this is not the case and it has to be Joomla Project in the author field.

**Comment by conconnl** [comment]
I don't know what is right or wrong.
We are talking about the XML file in the Core Installer, so not Language Packages.
Personally I can imagine it's the same copyright as all other Core files

**Comment by ot2sen** [comment]
Then it has changed without updating the Translation policy.
See section 'Licenses' point 2:
https://docs.joomla.org/Translation_of_Language_Packs

**Comment by zero-24** [comment]
> Translations of the installation files that are contributed for release in the official Joomla core release are derivative works and must be provided under GNU GPL. Copyrights are both Open Source Matters and author's. This work is acknowledged in the main credit file of the Joomla CMS. The XML file for the installation may contain author's name. The description should only contain the name of the language in English or local language.

So we are chnaging it back again?

**Comment by conconnl** [comment]
> Then it has changed without updating the Translation policy.
> See section 'Licenses' point 2:
> https://docs.joomla.org/Translation_of_Language_Packs

In my opinion those docs have never been really maintained.
Last edit was almost 5 years ago....
If we are told how it should be by the ones who wrote the document, as they are now saying the current files are wrong, they should have updated the document they wrote.

I see it very simple.
Or it's changed and the Document as well.
Or people should stop complaining that things are wrong, while documentation says anything else.

I won't press merge anymore....
The GitHub maintainers can do this if the decision is made.

File: Configurations/Crowdin-J4-Installer.yml [hunk 1]
```diff
@@ -22,11 +22,6 @@ files:
     type: xml
     translatable_elements:
       - /metafile/name
```
Comment (zero-24) [inline Configurations/Crowdin-J4-Installer.yml:24]:
```suggestion
      - /metafile/name
      - /metafile/author
```

**Comment by zero-24** [comment]
PR for German has been done: https://github.com/joomla/core-translations/pull/23 and suggestion for this here has been added.

Please review and merge :)

**Comment by HLeithner** [comment]
I'm happy following the documentation. The problem is only that more or less "all" (ok many) xml files in all/many translations are wrong in some kind... even in the core.

I add the author translation for crowdin.

**Comment by brianteeman** [comment]
> We (German) have been told this is not the case and it has to be Joomla Project in the author field.

Are you sure that it wasn't the copyright field you were told to change as you had removed the OSM copyright and replaced it with your own

**Comment by zero-24** [comment]
We have been contacted and beeing told to change both the author and the copyright. Its now reverted back so all is good now :)

**Pull request merged**